<launch>

    <!--  ************** Global Parameters ***************  -->
    <param name="/use_sim_time" value="true"/>

    <!--  ************** Create world file ***************  -->
    <!-- <rosparam command="load" file="$(find dwa_multi_robot)/config/random_scenarios.yaml"/> -->
    <!-- <node pkg="dwa_multi_robot" type="random_scenarios" name="random_scenarios" output="screen"/> -->
    <!--  ************** Stage Simulator ***************  -->
    <node pkg="stage_ros" type="stageros" name="dwa_envrionment" args="$(find dwa_multi_robot)/world/hospital_section.world">
        <remap from="base_scan" to="scan"/>
    </node>
</launch>
